To: vim_dev@googlegroups.com
Subject: Patch 7.3.416
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.416 (after 7.3.415)
Problem:    Compiler warning for wrong pointer.
Solution:   Add type cast.
Files:	    src/eval.c


*** ../vim-7.3.415/src/eval.c	2012-01-26 14:32:26.000000000 +0100
--- src/eval.c	2012-01-26 15:42:56.000000000 +0100
***************
*** 21802,21808 ****
  	fp = HI2UF(hi);
  
  	if (fp->uf_flags & FC_DICT)
! 	    return ""; /* don't show dict functions */
  
  	if (STRLEN(fp->uf_name) + 4 >= IOSIZE)
  	    return fp->uf_name;	/* prevents overflow */
--- 21802,21808 ----
  	fp = HI2UF(hi);
  
  	if (fp->uf_flags & FC_DICT)
! 	    return (char_u *)""; /* don't show dict functions */
  
  	if (STRLEN(fp->uf_name) + 4 >= IOSIZE)
  	    return fp->uf_name;	/* prevents overflow */
*** ../vim-7.3.415/src/version.c	2012-01-26 14:32:26.000000000 +0100
--- src/version.c	2012-01-26 15:43:31.000000000 +0100
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     416,
  /**/

-- 
It's totally unfair to suggest - as many have - that engineers are socially
inept.  Engineers simply have different objectives when it comes to social
interaction.
				(Scott Adams - The Dilbert principle)

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
